core/includes/drupal.inc

```diff
@@ -2550,23 +2549,72 @@ function system_settings_form($form) {
  * forms, developers should create separate submit handlers for each form
  * to save the settings into various configuration files as needed.
  *
- * @deprecated since 1.0.0
  */
 function system_settings_form_submit($form, &$form_state) {
```

Comment:
If this function is no longer deprecated, it needs to be moved out of `drupal.inc`, otherwise it will only be available if the Drupal compatibility layer is enabled in settings.php (which it is by default, but many sites disable it).